standard/links/commons/links_match_priority_groups.lua

```diff
@@ -0,0 +1,22 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Links/MatchPriorityGroups
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	-- common
+	'vod',
+	'preview',
+	'interview',
+	'review',
+	'lrthread',
+	'recap',
+	'headtohead',
+
+	-- ageofempires
+	'mapdraft',
+	'civdraft'
+}
```

Comment:
Maybe have one group for matches stuff and one group for map stuff?

Reply:
How would we distinguish between them?